Chummer/frmCharacterOptions.cs

```diff
@@ -734,6 +782,7 @@ private void PopulateCustomDataDirectoryTreeView()
                     if (objCustomDataDirectory.Item1 is CustomDataDirectoryInfo objInfo)
                     {
                         objNode.Text = objInfo.Name;
+                        objInfo.LazyCreate();
```

Comment:
Shouldn't this be the other way around?

Reply:
You mean the lazy create should be before the other to assignments?
That is not really necessary right now, because `Name` and `Path` are normally set in the constructor.

But it may be clearer or better to read for others to move it up a bit.